programs/launcher/main.cpp

```diff
@@ -326,6 +339,7 @@ struct launcher_def {
   last_run_def last_run;
   int start_delay;
   bool nogen;
+  bool add_enable_stale_production;
```

Comment:
These uninitialized bools and ints worry me. add_enable_stale_production is not set unless mode is passed. Which means checking it is undefined and could crash or assume true or false.